egov/egov-egi/src/main/java/org/egov/infra/admin/master/service/UserService.java

```diff
@@ -180,14 +190,72 @@ else if (user.getType().equals(UserType.EMPLOYEE))
         createUserRequest.setRequestInfo(microserviceUtils.createRequestInfo());
 
         final RestTemplate restTemplate = new RestTemplate();
+        UserDetailResponse udr = null;
         try {
-            restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
+            udr = restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
         } catch (final Exception e) {
```

Comment:
Web related things should not be hooked to services, can you move this restTemplate create call to Controller?